pkg/build/node/import.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package node
+
+import (
+	"os"
+
+	"sigs.k8s.io/kind/pkg/exec"
+)
+
+type imageImporter interface {
+	Prepare() error
+	LoadCommand() exec.Cmd
+	End() error
+}
+
+type containerdImporter struct {
+	containerCmder exec.Cmder
+}
+
+func newContainerdImporter(containerCmder exec.Cmder) imageImporter {
+	return &containerdImporter{
+		containerCmder: containerCmder,
+	}
+}
+
+var _ imageImporter = &containerdImporter{}
+
+func (c *containerdImporter) Prepare() error {
+	if err := c.containerCmder.Command(
+		"bash", "-c", "nohup containerd > /dev/null 2>&1 &",
```

Comment:
we could probably rewrite this to just need sh but i'm not sure we care right now

Reply:
NOTE: we are writing this to /dev/null for now so it doesn't leave any stray files in the built image, but we should consider debuggability here in the future. Ideally viewing this isn't necessary.